src/electron/adapters/electron-storage-adapter.ts

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { IndexedDBDataKeys } from '../../background/IndexedDBDataKeys';
+import { StorageAdapter } from '../../common/browser-adapters/storage-adapter';
+import { IndexedDBAPI } from '../../common/indexedDB/indexedDB';
+
+export class ElectronStorageAdapter implements StorageAdapter {
+    constructor(private readonly indexedDBInstance: IndexedDBAPI) {}
```

Comment:
code seems a little cramped here... some new lines between functions may help (at this point, I'm not even sure why we don't have a tslint/prettier setting for this)